packages/core/package.json

```diff
@@ -4,6 +4,7 @@
   "description": "UMA smart contracts and unit tests",
   "dependencies": {
     "@eth-optimism/contracts": "^0.4.5",
+    "@eth-optimism/message-relayer": "^0.1.8",
```

Comment:
Do any of these need to be production dependencies? It seems like they are all only used in tests, right?

Reply:
this is true! will move it to the dev-dependencies!